types/expo/index.d.ts

```diff
@@ -2057,6 +2058,7 @@ export interface SvgCommonProps {
     strokeLineJoin?: string;
     strokeDasharray?: any[];
     strokeDashoffset?: any;
+    transform?: string | object;
```

Comment:
What do you think about specify the type of that `object` (e.g.  `string | { translateX?: number, translateY?: number }`)

Reply:
I'm just using the string variant transform="..." myself, so I would like to leave that as just "|object" for now. You can also do transform="skewX(30)" etc, and there is an equivalent in the object syntax as well so just doing ```{ translateX?: number, translateY?: number }``` wouldn't be entirely correct. To refine the object into a more specific type I would have to dig into exactly what members are available on the react-native-svg objects. I suspect it's scale,rotation,skew etc.